src/Symfony/Component/Notifier/Bridge/Smsmode/SmsmodeOptions.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Smsmode;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author gnito-org <https://github.com/gnito-org>
+ */
+final class SmsmodeOptions implements MessageOptionsInterface
```

Comment:
Class should be removed

Reply:
I'd much rather add an optional `MessageOptionsInterface` parameter to the `SmsMessage::__construct()` method.

Most of the options that I've defined cannot be set in the DSN because they can vary by message.

Reply:
We merged #48254, so lets keep it 👍